lib/groups.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+/**
+ * API Wrapper for the Gambit Groups API.
+ */
+
+const superagent = require('superagent');
+const logger = app.locals.logger;
+
+const uri = process.env.GAMBIT_GROUPS_API_URI;
+const apiKey = process.env.GAMBIT_GROUPS_API_KEY;
+const apiHeaderName = 'x-gambit-group-api-key';
+
+/**
+ * Make a GET request to Gambit Groups API.
+ * Handles authentication & URL formation.
+ * @param  {string} path API Path. eg: /group
+ * @return {Promise}
+ */
+function get(path) {
+  return superagent.get(`${uri}${path}`)
+    .set(apiHeaderName, apiKey)
+    .then(res => res.body)
+    .catch(error => logger.error(error));
+}
+
+/**
+ * Make a post request to Gambit Groups API.
+ * Handles authentication & URL formation.
+ * @param  {string} path API Path. eg: /group
+ * @param  {object} data JSON data to send.
+ * @return {Promise}
+ */
+function post(path, data) {
+  return superagent.post(`${uri}${path}`)
+    .send(data)
+    .set(apiHeaderName, apiKey)
+    .then(res => res.body)
```

Comment:
Wait. Why do you call then/catch here? Just return the promise. You're thenning it above anyways.